CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ This file is influenced by http://keepachangelog.com/.
 
 - Updated 'Popular now' links
 
+- Updated homepage category descriptions
```

Comment:
There shouldn't be a blank line between this and the previous dot point (I know that's how the others look but it still shouldn't have the blank line)